xml/deploy_additional.xml

```diff
@@ -300,6 +300,53 @@
       </listitem>
      </itemizedlist>
     </note>
+    <important>
+     <title>Incorrectly reported image sizes and the <option>emulate_legacy_capacity</option> option</title>
+     <para>
+      A bug in older kernels caused that an RBD target reported an image size
+      one block (512 bytes) larger than it actually was. Therefore, the LUN
+      could write one additional block of the image. To keep the image data
+      consistent, new kernels support the old behavior by default unless you
+      set <option>emulate_legacy_capacity</option> attribute to 0 for a target.
```

Comment:
This looks good. I'd suggest a minor rework to avoid confusion between iSCSI targets, LUNs and images:

```
A bug in older kernels caused iSCSI targets to report LUNs as one block
(512 bytes) larger than the underlying RBD image. This allowed iSCSI
initiators to write to one additional block of the image. To keep the image
data consistent, new kernels support the old behavior by default unless you
set the <option>emulate_legacy_capacity</option> attribute to 0.
```

- emulate_legacy_capacity is an LIO backstore (LUN) attribute, i.e. it doesn't
  apply to the iSCSI target IQN, which could have multiple LUNs. I'm not sure
  how the UI presents it, but it'd make sense to use the same terms.